terraform_compliance/extensions/terraform.py

```diff
@@ -255,7 +255,12 @@ def _find_resource_from_name(self, resource_name):
             module = self.raw['configuration']['root_module'].get('module_calls', {}).get(module_name, {})
 
             output_value = module.get('module', {}).get('outputs', {}).get(output_id, {})
-            resources = output_value.get('expression', {}).get('references') if 'expression' in output_value else output_value.get('value', [])
+            if 'expression' in output_value:
+                output_value = output_value.get('expression', {})
+                resources = output_value.get('references') if 'references' in output_value else []
+            else:
+                output_value.get('value', [])
+
```

Comment:
We can change this two line with ;

```python
resources = output_value.get('expression', {}).get('references', []) if 'expression' in output_value else output_value.get('value', [])
```

Reply:
So actually the change is just adding a `, []` to the second `get()` for `references` :)